README.md

```diff
@@ -11,3 +11,33 @@ For more information about how the admin container fits into the Bottlerocket op
 
 You'll need Docker 17.06.2 or later, for multi-stage build support.
 Then run `make`!
+
+## Authenticating with the Admin Container
+
+Starting from v0.6.0, users have the option to pass in their own ssh keys rather than the admin container relying on the AWS instance metadata service (IMDS).
+
+Users can add their own keys by populating the admin container's user-data with a base64-encoded JSON block.
```

Comment:
It's probably worth mentioning that if the user-data setting is populated for this container, we aren't going to fetch any keys.

Reply:
> It's probably worth mentioning that if the user-data setting is populated for this container, we aren't going to fetch any keys.

Not just not fetch any keys, but also not handle populating settings for **anything else** at this time - even if there aren't other pieces of data yet. If this is the expected handling (for the foreseeable future) then we should help users understand that they may be opting out of *any* populated settings (at least until that changes..).